packages/composites/token-allocation/src/config/index.ts

```diff
@@ -15,20 +15,21 @@ import * as Nomics from '@chainlink/nomics-adapter'
 import * as NCFX from '@chainlink/ncfx-adapter'
 import * as Tiingo from '@chainlink/tiingo-adapter'
 
+// TODO types
 export const adapters: AdapterImplementation[] = [
-  Amberdata,
-  CFBenchmarks,
-  CoinApi,
-  CoinGecko,
-  CoinMarketCap,
-  CoinMetrics,
-  CoinPaprika,
-  CryptoCompare,
-  Finage,
-  Kaiko,
-  NCFX,
-  Nomics,
-  Tiingo,
+  Amberdata as any,
+  CFBenchmarks as any,
+  CoinApi as any,
+  CoinGecko as any,
+  CoinMarketCap as any,
+  CoinMetrics as any,
+  CoinPaprika as any,
+  CryptoCompare as any,
+  Finage as any,
+  Kaiko as any,
+  NCFX as any,
+  Nomics as any,
+  Tiingo as any,
```

Comment:
Seems like we used `unknown` as an intermediary type in the past. Should we do that here as well?